test/functional/p2p_submit_orphan.py

```diff
@@ -71,7 +71,7 @@ def run_test(self):
         # Check both transactions have propagated to the last peer
         for node in self.nodes:
             has_txs = lambda: all(node.mempool_contains_tx(tx_id) for tx_id in [tx1_id, tx2_id])
-            self.wait_until(has_txs, timeout = 5)
+            self.wait_until(has_txs, timeout = 10)
```

Comment:
Please add a comment specifying that this number should be larger than so and so